### PR TITLE
SAK-32268 - Delegated Access needs date picker update

### PR DESCRIPTION
**Pull request by josecebe**

### CODE REVIEW

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/BasePage.java [hunk 1]
```diff
@@ -55,6 +56,7 @@
 public class BasePage extends WebPage implements IHeaderContributor {
 
 	private static final Logger log = LoggerFactory.getLogger(BasePage.class); 
+	private static ResourceLoader rloader = new ResourceLoader("sitenav");
```
Comment (mpellicer) [inline delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/BasePage.java]: "sitenav" should be removed.

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/EditablePanelDate.java [hunk 2]
```diff
@@ -58,32 +71,42 @@ protected void onComponentTag(ComponentTag tag) {
...
+						((BaseTreePage)target.getPage()).getTree().getTreeState().collapseNode(node);
+					}
+					((BaseTreePage)target.getPage()).getTree().updateTree(target);
+					target.focusComponent(hiddenInput);
 				}
-				((BaseTreePage)target.getPage()).getTree().updateTree(target);
 			}
-
 		});
+
```
Comment (mpellicer) [inline delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/EditablePanelDate.java]: Clean this empty line.

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/EditablePanelDate.java [hunk 3]
```diff
@@ -58,32 +71,42 @@ protected void onComponentTag(ComponentTag tag) {
...
 		add(date);
+		add(hiddenInput);
 	}
-
```
Comment (mpellicer) [inline delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/EditablePanelDate.java]: Clean this empty line

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/ShoppingEditBulkPage.java [hunk 4]
```diff
@@ -166,9 +168,7 @@ public void clicked(AjaxRequestTarget target) {
 		};
 		deleteTable.setOutputMarkupId(true);
 		form.add(deleteTable);
-
```
Comment (mpellicer) [inline delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/ShoppingEditBulkPage.java]: Clean this empty lines

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/ShoppingEditPage.java [hunk 5]
```diff
@@ -208,7 +216,6 @@ protected void onSubmit(AjaxRequestTarget target, Form<?> arg1) {
 			}
 		});
 		
-
```
Comment (mpellicer) [inline delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/tool/pages/ShoppingEditPage.java]: Clean this empty line.

File: delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/utils/DateUtil.java [hunk 6]
```diff
@@ -0,0 +1,105 @@
+/**********************************************************************************
+ * $URL$
+ * $Id$
+ **********************************************************************************
+ *
+ * Copyright (c) 2008 The Sakai Foundation
```
Comment (mpellicer) [inline delegatedaccess/tool/src/java/org/sakaiproject/delegatedaccess/utils/DateUtil.java]: Year 2017

**Comment by mpellicer** [comment]
@josecebe please rebase this PR to keep it updated.

Thanks.

**Comment by josecebe** [comment]
Done @mpellicer

**Pull request merged**